charts/node/values.yaml

```diff
@@ -227,6 +227,9 @@ node:
   #   seed: "//Alice"
   #   extraDerivation: //babe
 
+  ## Inject keys from already existing kubernetes secrets
+  keysExistingSecrets: []
```

Comment:
Can this be renamed to `node.existingSecrets.keys` to follow the same convention as `node.vault.keys`.
Can you also implement `node.existingSecrets.nodeKey` for loading the node key from a secret. This can be a more secure alternative to `node.customNodeKey` which passes the key as plain text.

Reply:
Hi @PierreBesson , that's a good idea, added this & changed naming. I also bumped version in Chart.yaml